src/test/java/pro/sky/sky/pro/CourseNumber2/JavaQuestionServiceSingleTest.java

```diff
@@ -0,0 +1,42 @@
+package pro.sky.sky.pro.CourseNumber2;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import pro.sky.sky.pro.CourseNumber2.Data.Question;
+import pro.sky.sky.pro.CourseNumber2.Exception.DataNotFound;
+import pro.sky.sky.pro.CourseNumber2.Exception.NotNullDataException;
+import pro.sky.sky.pro.CourseNumber2.Service.JavaQuestionService;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class JavaQuestionServiceSingleTest {
+
+    static final JavaQuestionService javaQuestionService  = new JavaQuestionService();
+
+    @BeforeEach
+    public void setUp(){
```

Comment:
```suggestion
    @BeforeAll
    public static void setUp(){
```
иначе мы пытаемся дважды добавить поля в сервис